README.md

```diff
@@ -0,0 +1,90 @@
+# AILuminate v1.0 DEMO Prompt Set
+
+This file contains the DEMO prompt library of the AILuminate 1.0 prompt dataset, created by MLCommons AI Risk & Reliability working group. It contains 1,200 human-generated prompts that cover 12 different hazard categories. The dataset is a mix of adversarial and non-adversarial prompts. This dataset is a 10% subset of the full AILuminate training dataset.
+
+# Background
+
+The MLCommons [AILuminate v1.0 benchmark](https://mlcommons.org/ailuminate/) provides safety testing for general purpose chat systems across 12 hazard categories and includes:
+
+* A **safety assessment standard** including a hazard taxonomy and response evaluation criteria.  
+* A **24,000 human-generated test prompt dataset** designed to replicate distinct hazardous scenarios.  
+  * 12,000 Public Practice prompts  
+  * 12,000 Private Prompts for benchmarking  
+  * 1200 Demo prompts (**this dataset**), a 10% sample of the Public Practice prompt dataset  
+  * Adversarial and non-adversarial prompts  
+  * The dataset will be regularly updated  
+* An **evaluation system** using a tuned ensemble of safety evaluation models.  
+* [Modelbench](https://github.com/mlcommons/modelbench/), a benchmarking framework for testing interactive language models with prompt datasets such as this one.  
+* The dataset is only in American English, with additions of other languages (French, Simplified Chinese, and French) in early 2025\. Support for other languages is planned.
+
+![][image1]
+
+# Dataset overview
+
+This dataset contains 1,200 human-generated prompts that cover 12 different hazard categories.
+
+## Hazard definition and assessment standard
+
+See [AILuminate 1.0 Assessment Standards](https://drive.google.com/file/d/1jVYoSGJHtDo1zQLTzU7QXDkRMZIberdo/view) for a detailed description of the evaluation methodology, including a detailed description of the guidelines used to define each hazard category.
+
+## Hazard categories
+
+| Content Hazard Categories |  |
```

Comment:
This table ends up pretty weird looking on Github, so I'd suggest rendering it in a way that's more straightforward.